freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaChecks.java

```diff
@@ -7,7 +7,9 @@
     /* Users in this list should not be added/deleted. */
     public static final List<String> IPA_PROTECTED_USERS = List.of("admin");
 
-    /* Groups in this list should not be added/deleted. */
+    /* Groups in this list should not be added/deleted. These groups are considered to be reserved in CDP-UMS.
```

Comment:
This comment is now ambiguous because we are now also referring to the UMS (instead of just FreeIPA). Can you update these comments to say that "Groups in this list should not be added to or deleted from FreeIPA".
Also change the above comment to "Users in this list should not be added to or deleted from FreeIPA"
Also change the below comment to "Group membership in this groups should not be explicitly changed in FreeIPA"

Reply:
Done.